Automattic/wp-calypso
# Domains: Update illustration on G Suite offer

This PR updates the G Suite offer screen to use the new illustration style. There were also some concerns about possibly infringing on Google's trademark by using the colorful G logo (as stated in their [branding guidelines](https://docs.google.com/presentation/d/1woCMr2x9BnSOuDbfYtghC2VEJI7dHxsRLYq1v4p-DDY/edit#slide=id.g1e6d2a35db_0_63)).

<img width="1069" alt="screen shot 2017-06-26 at 11 33 12" src="https://user-images.githubusercontent.com/448298/27547691-c3697196-5a64-11e7-8c86-df57a86ae027.png">

This also fixes some weird spacing around the illustration at the breakpoint between `960px` and `1040px`.

Before | After
------------ | -------------
<img width="666" alt="screen shot 2017-06-26 at 11 42 52" src="https://user-images.githubusercontent.com/448298/27549456-019987f8-5a6a-11e7-9671-e85699e50bee.png"> | <img width="649" alt="screen shot 2017-06-26 at 12 14 53" src="https://user-images.githubusercontent.com/448298/27549450-fe0eaeba-5a69-11e7-804e-17596651579c.png">

#### Testing Instructions
1. Checkout `update/g-suite-illustration` or open a [live branch](https://calypso.live/?branch=update/g-suite-illustration)
2. Go to a site that has a custom domain but no email through G Suite
3. Click `Domains > Email` or visit `http://calypso.localhost:3000/domains/manage/email/:domain`
4. Assert the new illustration is shown as above.
